modApiServer/src/org/aion/api/server/pb/ApiAion0.java

```diff
@@ -2072,9 +2072,7 @@ public ApiAion0(IAionChain ac) {
 
                     // blocks requested in the future. return empty result
                     if (blkStart > latestBlkNum) {
-                        Message.rsp_getBlockDetailsByRange rsp =
-                            Message.rsp_getBlockDetailsByRange
-                                .newBuilder()
+                        Message.rsp_getBlockDetailsByRange rsp = Message.rsp_getBlockDetailsByRange.newBuilder()
```

Comment:
do you use google java formatter to format the code? the ApiAion0 looks no code has been changed. can you rever the change if this PR has no related with ApiAion0?

Reply:
The Google formatting changes made that function in ApiAion0 pretty unreadable so these formatting changes were just done so I could read that implementation. Will revert these changes now since they are un-related to this PR.

Reply:
Ok, removed all changes to file ApiAion0